packages/api/lib/writeRecords/write-granules.js

```diff
@@ -882,7 +913,7 @@ const writeGranulesFromMessage = async ({
       }
 
       const apiGranuleRecord = await generateGranuleApiRecord({
-        granule: { ...granule, published },
+        granule: { ...granule, published, createdAt: granule.createdAt || workflowStartTime },
```

Comment:
Yup so we prefer the granule's existing `createdAt` because if this is a new workflow run on a previously ingested granule, we want to preserve the original `createdAt`. Otherwise look at the workflow's start time.

Reply:
Exactly - this is a consequence of the other workflow/default logic moves. 

Also missed 523f66deb3